WireNetwork/Sources/WireNetwork/Authentication/CookieStorage.swift

```diff
@@ -163,8 +163,8 @@ public actor CookieStorage: CookieStorageProtocol {
     }
 
     private func updateCookieInKeychain(_ cookieData: Data) async throws {
-        let updateQuery = updateQuery(cookieData: cookieData)
-        try await keychain.updateItem(query: fetchQuery, attributesToUpdate: updateQuery)
+        let updateQuery: Set<KeychainQueryItem> = [.data(cookieData.base64EncodedData())]
+        try await keychain.updateItem(query: baseQuery, attributesToUpdate: updateQuery)
```

Comment:
This fixes another bug where updating the cookie failed. The fix is to use simply the base query to identify the object needing update, and the update query should only contain the data to update.